core/src/interactive.rs

```diff
@@ -457,6 +457,7 @@ impl BroadcastMessage {
             PutDownCards => format!("{} put down the bottom cards", n?),
             GameFinished { result: _ } => "The game has finished.".to_string(),
             BonusLevelEarned => "Landlord team earned a bonus level for defending with a smaller team".to_string(),
+            GamesetWinnerAnnoucement {ref player_name} => format!("{} won the game set!", player_name),
```

Comment:
"{} successfully defended on A!", instead? "game set" isn't (as far as I know) a common term in the game.

Reply:
I just google up other card games. Seems like the most common way to describe trick taking games is to name the entire series as one game, and each "play" as one deal. so winners win the game by advancing their ranks/levels in each deal until A is successfully defended. Since game is used in place of deal, it is bit hard to come up with a common term. 

"successfully defended on A" does not carry the significance of winning the game, I think. 

"{} won the game series"? or simply "Congratulations! {} is the winner!"?